test/parallel/test-crypto-dh.js

```diff
@@ -258,10 +270,10 @@ ecdh5.setPrivateKey(cafebabeKey, 'hex');
   '0000000000000000000000000000000000000000000000000000000000000000',
   'FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364141',
   'FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF',
-].forEach(function(element, index, object) {
-  assert.throws(function() {
+].forEach((element, index, object) => {
```

Comment:
`index` and `object` are not used; they were excluded in https://github.com/nodejs/node/pull/10753 (didn't realize this PR existed). Can you remove?